kubetest2-gce/README.md

```diff
@@ -0,0 +1,19 @@
+# Kubetest2 GCE Deployer
+
+This component of kubetest2 is responsible for test cluster lifecycles for clusters deployed to GCE VMs.
+
+The [original design proposal](https://docs.google.com/document/d/157nSQNyy9cOjw4izG0rUs_9z9Suy31JQtng_g2peTsw/edit#heading=h.5irk4csrpu0y) has a great deal of detail about the motivations for the deployer and detailed evaluations of the original (kubetest) deployer it replaces.
+
+## Usage
+
+Currently, the GCE deployer must be running on a system with a version of k/k or the cloud-provider-gcp repository cloned. A simple run without running tests looks as follows:
```

Comment:
maybe add some links to those repos and add a line about why, i.e. we still use kube-up to do the actual cluster creation.